tests/conftest.py

```diff
@@ -526,8 +585,9 @@ def tpch_cluster(request, dask_env_variables, cluster_kwargs, github_cluster_tag
         **cluster_kwargs["tpch"],
     )
     dump_cluster_kwargs(kwargs, f"tpch.{module}")
-    with Cluster(**kwargs) as cluster:
-        yield cluster
+    with dask.config.set({"distributed.scheduler.worker-saturation": "inf"}):
```

Comment:
I'm a bit confused. This should be on main already, isn't it?

Reply:
Ok, I think this is a rendering issue https://github.com/coiled/benchmarks/blob/b8f0f0ee44fcd4d92c1e244a8b4c6c709ae3079f/tests/conftest.py#L518-L531